IDR/notebook/0.get_ids.ipynb

```diff
@@ -12,7 +12,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 11,
```

Comment:
For notebooks to be merged, they need to be run in _sequential order_. i.e. this execution count should be = 1

Reply:
I'm not sure how that happened... I went back and fixed the execution order of each code chunk. Although, there is an "invisible" execution in `0.get_ids.ipynb` at the end which is executed as number 7. So it looks like `0.get_ids.ipynb` ends at 6 and `1.describe_screens.ipynb`  begins at 8.

Added in cffe6209681ec4afcf3a6f46ed00737bd74543f5.